app/code/Shiphawk/Order/etc/adminhtml/system.xml

```diff
@@ -19,14 +19,18 @@
                     <label>Order Sync</label>
                     <source_model>Shiphawk\Order\Model\System\Configuration\Source\Ordersync</source_model>
                 </field>
-                <field id="shiphawk_api_key" translate="label" type="obscure" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="0">
+                <field id="shiphawk_api_key" translate="label" type="password" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="0">
```

Comment:
This change was made because when he have saved Api Key, and then edit, for example, "Gateway Url", Api Key value will be changed to "******". In previous implementation we need to enter Api Key every time we edit ShipHawk Order Options form.